src/styles/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: left;
+  margin-bottom: 16px;
+  color: var(--secondary-color);
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-counter {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--secondary-color);
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-color);
+}
+
+.card__button {
+  display: flex;
+  text-align: center;
+  padding: 11px 70px 11px 69px;
```

Comment:
better to give it `width: 100%` so that you don't have to hardcode horizontal paddings